### PR TITLE
Cursor: Draw on Dual Screen

### PR DESCRIPTION
**Pull request by badosu**

### CODE REVIEW

File: rts/Game/UI/MouseCursor.h [hunk 1]
```diff
@@ -41,7 +41,12 @@ class CMouseCursor {
 	const std::string& GetName() const { return name; }
 	const HotSpot GetHotSpot() const { return hotSpot; }
 
-	float4 CalcFrameMatrixParams(const float3& winCoors, const float2& winScale) const;
+	float4 CalcFrameMatrixParams(const float3& winCoors, const float2& winScale) const
+	{
+		return CalcFrameMatrixParams(winCoors, winScale, false);
+	}
+
+	float4 CalcFrameMatrixParams(const float3& winCoors, const float2& winScale, const bool fullWindow) const;
```
Comment (sprunk) [inline rts/Game/UI/MouseCursor.h:49]: How about a default arg?
```suggestion
	float4 CalcFrameMatrixParams(const float3& winCoors, const float2& winScale, const bool fullWindow = false) const;
```

Comment (badosu) [reply rts/Game/UI/MouseCursor.h:49]: Isn't that `fullWindow = false`? Or is there another way for default args in cpp?

Comment (badosu) [reply rts/Game/UI/MouseCursor.h:49]: Ah, I'm stupid lol

**Comment by badosu** [comment]
Didn't test, am not interested in pushing this work forward. Anybody interested is welcome to fork the branch and proceed as they desire.

**Pull request closed without merging**